godot-core/src/builtin/aabb.rs

```diff
@@ -200,7 +200,7 @@ impl Aabb {
     /// Returns the index of the longest axis of the AABB (according to Vector3's AXIS_* constants).
     #[inline]
     pub fn longest_axis_index(&self) -> Vector3Axis {
-        self.size.max_axis_index()
+        self.size.max_axis().unwrap_or(Vector3Axis::X)
     }
```

Comment:
I wonder if we should make those return `Option<Vector3Axis>` as well and document how the `unwrap_or` could be applied?

Simply because it's very easy to unwrap, but it's harder to differentiate whether `Vector3Axis::X` means it's the longest or there are multiple equally long ones.

Reply:
Should I also change `longest_axis()` return type to `Option<Vector3>`?

Reply:
That would be great, yes! Thanks :slightly_smiling_face: